gector/gec_model.py

```diff
@@ -149,46 +167,61 @@ def predict(self, batches):
     def get_token_action(self, token, index, prob, sugg_token):
         """Get lost of suggested actions for token."""
         # cases when we don't need to do anything
-        if prob < self.min_error_probability or sugg_token in [UNK, PAD, '$KEEP']:
+        if prob < self.min_error_probability or sugg_token in [
+            UNK,
+            PAD,
+            "$KEEP",
+        ]:
             return None
 
-        if sugg_token.startswith('$REPLACE_') or sugg_token.startswith('$TRANSFORM_') or sugg_token == '$DELETE':
+        if (
+            sugg_token.startswith("$REPLACE_")
+            or sugg_token.startswith("$TRANSFORM_")
+            or sugg_token == "$DELETE"
+        ):
             start_pos = index
             end_pos = index + 1
-        elif sugg_token.startswith("$APPEND_") or sugg_token.startswith("$MERGE_"):
+        elif sugg_token.startswith("$APPEND_") or sugg_token.startswith(
+            "$MERGE_"
+        ):
             start_pos = index + 1
             end_pos = index + 1
 
         if sugg_token == "$DELETE":
             sugg_token_clear = ""
-        elif sugg_token.startswith('$TRANSFORM_') or sugg_token.startswith("$MERGE_"):
+        elif sugg_token.startswith("$TRANSFORM_") or sugg_token.startswith(
+            "$MERGE_"
+        ):
             sugg_token_clear = sugg_token[:]
         else:
-            sugg_token_clear = sugg_token[sugg_token.index('_') + 1:]
+            sugg_token_clear = sugg_token[sugg_token.index("_") + 1 :]
 
         return start_pos - 1, end_pos - 1, sugg_token_clear, prob
 
     def _get_embbeder(self, weigths_name, special_tokens_fix):
-        embedders = {'bert': PretrainedBertEmbedder(
-            pretrained_model=weigths_name,
-            requires_grad=False,
-            top_layer_only=True,
-            special_tokens_fix=special_tokens_fix)
+        embedders = {
+            "bert": PretrainedBertEmbedder(
+                pretrained_model=weigths_name,
```

Comment:
maybe we should correct `weigths_name` to `weights_name`

Reply:
Yeah, it's a real eyesore.